models/cow_protocol/tx_hash_labels/stable_to_stable/ethereum/cow_protocol_tx_hash_labels_stable_to_stable_ethereum.sql

```diff
@@ -0,0 +1,37 @@
+{{
+    config(
+        alias='stable_to_stable_ethereum',
+    )
+}}
+
+with
+ stable_to_stable_trades as (
+    select
+        distinct tx_hash
+    from {{ ref('cow_protocol_ethereum_trades') }}
+    where token_pair in (
+            'USDT-DAI',
+            'USDT-USDC',
+            'USDT-BUSD',
+            'DAI-USDT',
+            'DAI-USDC',
+            'DAI-BUSD',
+            'USDC-USDT',
+            'USDC-DAI',
+            'USDC-BUSD',
+            'BUSD-USDC',
+            'BUSD-USDT',
+            'BUSD-DAI'
+        )
```

Comment:
Maybe you forgot to commit the changes here, but this is still using the token symbols.

Reply:
Thanks for pointing this out. I've updated the spell and implemented the `tokens_ethereum_erc20_stablecoins` table.